src/main/java/org/jala/efeeder/order/OrderCommand.java

```diff
@@ -22,43 +22,19 @@
     @Override
     public Out execute(In parameters) throws Exception {
 
-        List<Order> orders = new ArrayList<>();
-
         Out out = new DefaultOut();
         Connection connection = parameters.getConnection();
-        PreparedStatement preparedStatement = connection.prepareStatement(
-                "Select o.id_food_meeting, o.order_name, o.cost, u.id, u.email, u.name, u.last_name " +
-                "from orders o, user u where o.id_food_meeting = ? and o.id_user = u.id");
-        preparedStatement.setInt(1, Integer.valueOf(parameters.getParameter("id_food_meeting")));
-        ResultSet resultSet = preparedStatement.executeQuery();
-
-        while (resultSet.next()) {
-            orders.add(new Order(resultSet.getInt(1), new User(resultSet.getInt(4), resultSet.getString(5),
-                    resultSet.getString(6), resultSet.getString(7)), resultSet.getString(2), resultSet.getDouble(3)));
-        }
 
         List<User> users = new ArrayList<>();
-        preparedStatement = connection.prepareStatement("Select id, email, name, last_name from user");
-        resultSet = preparedStatement.executeQuery();
+        PreparedStatement preparedStatement = connection.prepareStatement("Select id, email, name, last_name from user");
```

Comment:
As we continue working on the code we can start moving this magic strings to static final variables